napari/_qt/layer_controls/qt_points_controls.py

```diff
@@ -252,7 +247,7 @@ def change_out_of_slice(self, state):
         state : QCheckBox
             Checkbox indicating whether to render out of slice.
         """
-        self.layer.out_of_slice_display = state == Qt.Checked
```

Comment:
Is this change required for Qt6?

Optional: I personally prefer the explicit comparison rather the truthy cast that would also make `Qt.PartiallyChecked => True` even though we shouldn't ever get that value right now.

Reply:
> the truthy cast that would also make Qt.PartiallyChecked => True

yeah, I think that's what was actually necessary here.  If I revert this change, I get a test fail on Qt6.  I've changed it to `state != Qt.Unchecked`

Reply:
actually, looks like even `state != Qt.Unchecked` doesn't work.  it _must_ be cast to bool to work on Qt6.   I suppose they've overloaded the `==` operator to return something that is not a bool in qt6?

Reply:
My only other thought is that Qt6 might require `state != Qt.CheckState.Unchecked`, but I don't have any evidence for that. Using the truthy cast is fine.